corehq/apps/api/tests/internal_resources.py

```diff
@@ -7,8 +7,8 @@
 class InternalTestMixin(object):
     def assert_accessible_via_sessions(self, url):
         # api auth should succeed
-        api_url = self._api_url(url, self.username)
-        response = self.client.get(api_url)
+        headers = self._get_api_key_auth_headers()
```

Comment:
What does this change do?

Reply:
this uses the `Authorization` header instead of passing the username and api key in URL params.